File: hw1/hw1.py
from typing import List

def main()  ->  None:
    encryptedMessage: List[str] = []

    shift: int = int(input("What is your shift (ints only)? "))
    message: str = input("What is your message? ")

    char: str
    for char in message:
        if char == " ":
            encryptedMessage.append(char)
            continue
        unicodeValue: int = ord(char)
        unicodeValue += shift
        encryptedMessage.append(chr(unicodeValue))

    print(f"""Encrypted Message:
    {"".join(encryptedMessage)}""")




if __name__ == "__main__":
    main()

File: hw1/hw1.cpp
#include <iostream>
#include <string>

using namespace std;

int main()  {
    int shift;
    string message;

    cout << "What is your shift (ints only)? ";
    cin >> shift;

    cout << "What is your message? ";
    cin.ignore();
    getline(cin, message);

    for (int i; i < message.length(); i++)  {
        char letter = message[i];
        if (letter == 32)   {
            cout << letter;
            continue;
        }

        cout << char(letter + shift);
    }

        return 0;
}
